src/Entity/OgMembership.php

```diff
@@ -137,13 +138,35 @@ public function getGroupEntityType() {
     return $this->get('entity_type')->value;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getGroupBundle() {
+    return $this->get('entity_bundle')->value;
+  }
+
   /**
    * {@inheritdoc}
    */
   public function getGroupId() {
     return $this->get('entity_id')->value;
   }
 
+  /**
+   * Checks if a group has already been populated on the membership.
+   *
+   * The group is required for a membership, so it is always present if a
+   * membership has been saved. This is intended for internal use to verify if
+   * a group is present when methods are called on a membership that is possibly
+   * still under construction.
```

Comment:
Maybe add a comment about performance, something like. ```In order to avoid loading the group just for this purpose, we access the `entity_type` property that should give us the same indication, but with a lower performance cost.```